apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { CoinBalance } from '~/ui/CoinBalance';
+import { Heading } from '~/ui/Heading';
+import { SenderRecipientAddress } from '~/ui/SenderRecipientAddress';
+
+type Recipient = {
+    address: string;
+    coin?: {
+        amount: number | string;
+        symbol?: string;
+    };
+};
+
+export interface SenderRecipientProps {
+    sender: string;
+    transferCoin?: boolean;
+    recipients?: Recipient[];
+}
+
+export function SenderRecipient({
+    sender,
+    recipients,
+    transferCoin,
+}: SenderRecipientProps) {
+    const multipleRecipients = recipients && recipients?.length > 1;
```

Comment:
FWIW you shouldn't need the `recipients &&`, the optional chain should take care of that for you.